docs/Animations.md

```diff
@@ -7,24 +7,270 @@ permalink: docs/animations.html
 next: accessibility
 ---
 
-Fluid, meaningful animations are essential to the mobile user
-experience. Animation APIs for React Native are currently under heavy
-development, the recommendations in this article are intended to be up
-to date with the current best-practices.
+Fluid, meaningful animations are essential to the mobile user experience. Like
+everything in React Native, Animation APIs for React Native are currently under
+development, but have started to coalesce around two complimentary systems:
+`LayoutAnimation` for animated global layout transactions, and `Animated` for
+more granular and interactive control of specific values.
+
+### Animated ###
+
+The `Animated` library is designed to make it very easy to concisely express a
+wide variety of interesting animation and interaction patterns in a very
+performant way. `Animated` focuses on declarative relationships between inputs
+and outputs, with configurable transforms in between, and simple `start`/`stop`
+methods to control time-based animation execution.  For example, a complete
+component with a simple spring bounce on mount looks like this:
+
+```javascript
+class Playground extends React.Component {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      bounceValue: new Animated.Value(1.5),   // Start large
+    };
+  }
+  render(): ReactElement {
+    return (
+      <Animated.Image                         // Base: Image, Text, View
+        source={{uri: 'http://i.imgur.com/XMKOH81.jpg'}}
+        style={{
+          flex: 1,
+          transform: [                        // `transform` is an ordered array
+            {scale: this.state.bounceValue},  // Map `bounceValue` to `scale`
+          ]
+        }}
+      />
+    );
+  }
+  componentDidMount() {
+    Animated.spring(                          // Base: spring, decay, timing
+      this.state.bounceValue,                 // Animate `bounceValue`
+      {
+        toValue: 0.8,                         // Animate to smaller size
+        friction: 1,                          // Bouncier spring
+      }
+    ).start();                                // Start the animation
+  }
+}
+```
+
+`bounceValue` is initialized to 1 as part of `state` in the constructor, and
+mapped to the scale transform on the image.  Behind the scenes, the numeric
+value is extracted and used to set scale.  When the component mounts, a spring
+animation is started on `bounceValue` which will update all of it's dependent
+mappings on each frame as the spring animates (in this case, just the scale).
+This is done in an optimized why that is faster than calling setState and
```

Comment:
"in an optimized way why" => "in an optimized way" or "in an optimized way, which is why"